src/webviews/webview-side/interactive-common/variableExplorer.tsx

```diff
@@ -348,6 +348,14 @@ export class VariableExplorer extends React.Component<IVariableExplorerProps, IV
                 } else if (variable.count) {
                     newSize = variable.count.toString();
                 }
+                let value = variable.value;
+                if (variable.type === 'str' && variable.value) {
+                    if (variable.value[0] === ' ') {
+                        value = `'${variable.value.slice(1)}'`;
+                    } else {
+                        value = `'${variable.value}'`;
+                    }
+                }
```

Comment:
What part of the code would add an empty space at the beginning of all of the received values?